### ISSUE TITLE
Animation frames origin points switching back to default

### ISSUE THREAD

**Issue by Marlin-dev** [Scirra/Construct-3-bugs#6352]
## Problem description
Origin point for animation frames reset to default after clicking on other frames in the animation

## Steps to reproduce

1. Open sprite in Animation Editor
2. Import files or strip.
3. Click on a frame of an animation (with multiple frames) and set the origin point to desired position.
4. Assign origin point to whole animation OR just click on another frame in that animation to set it's position.

## Observed result

 After step 3, click on any other frame and all origin points for all frames in that entire animation have been reset to default position (center of frame).

## Expected result

All origin points stay where they have been assigned.

## More details

A workaround I have found:  Set an image/origin point on one frame and immediately exit the Animation Editor. Save the game file and re-enter the Animation Editor for that same sprite. At this point, the origin points are in the newly assigned positions.

I tried to recreate this occurrence in a new file but was unable to trigger the bug. I would guess that this happens in approximately 5 out of 10 projects that I create. I thought it might be occurring when I import sprites from a strip, but my recent test project could not replicate the bug.

**Affected browsers/platforms:** 
I have only tested this issue on Chrome.

Currently using r317.2
I have noticed this issue for several stable versions.

**First affected release:** 

## System details

<details><summary>View details</summary>
Platform information
Product: Construct 3 r317.2 (stable)
Browser: Chrome 108.0.5359.125
Browser engine: Chromium
Context: browser
Operating system: Windows 10
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 12
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36
Language setting: en-US

Local storage
Storage quota (approx): 1.1 tb
Storage usage (approx): 34 mb (0%)
Persistant storage: No

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Google Inc. (NVIDIA)
Renderer: ANGLE (NVIDIA, NVIDIA GeForce RTX 2060 SUPER Direct3D11 vs_5_0 ps_5_0, D3D11)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_color_buffer_half_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_compression_bptc
EXT_texture_compression_rgtc
EXT_texture_filter_anisotropic
EXT_texture_norm16
KHR_parallel_shader_compile
OES_draw_buffers_indexed
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
WEBGL_multi_draw
OVR_multiview2
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

WebM Opus (audio/webm; codecs=opus)
Ogg Opus (audio/ogg; codecs=opus)
WebM Vorbis (audio/webm; codecs=vorbis)
Ogg Vorbis (audio/ogg; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM AV1 (video/webm; codecs=av01.0.00M.08)
MP4 AV1 (video/mp4; codecs=av01.0.00M.08)
WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.265 (video/mp4; codecs=hev1.1.2.L93.B0)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
PASTE HERE

</details>

**Comment by DiegoScirra** [comment]
Can't reproduce in either the latest stable nor latest beta. Tried importing different strips and replace the entire animation multiple times, as well as with multiple animations, everything seems to be working as expected.

Will keep this open for a while in case there is no more information, but if not will have to close.

**Comment by DiegoScirra** [comment]
Have a feeling nothing new is going to come up for this one, so I will close it.

@Marlin-dev You can always open a new issue if you have any new information.

**Comment by gutsalovpavel** [comment]
It is still happening

**Comment by IceFerretStudio** [comment]
sometimes when cropping sprites the origin gets set on the wrong place. I don't know what causes it since there is no indication that something is different from one sprite from another and it doesn't happen all the time. But when it happens i can't crop the sprite no matter what i try and it is not good since i need to crop hundreds of sprites.
